### PR TITLE
SNAP-984 Added column and links to name

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- [Added id column and make the name as link](https://osi-cwds.atlassian.net/browse/SNAP-984)

## Description
Added id column and make the name as link.

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/common/search/SearchResultsTable.jsx [hunk 1]
```diff
@@ -5,12 +5,20 @@ import 'react-table/react-table.css'
 
 class SearchResultsTable extends Component {
   columns = [{
+    Header: '',
+    id: 'row',
+    maxWidth: 50,
+    filterable: false,
+    Cell: (row) => {
+      return <div>{row.index + 1}.</div>
```
Comment (ggomez-cambriasolutions) [inline app/javascript/common/search/SearchResultsTable.jsx]: I don't think we need to wrap this in a div.

File: app/javascript/common/search/SearchResultsTable.jsx [hunk 2]
```diff
@@ -5,12 +5,20 @@ import 'react-table/react-table.css'
...
+    },
+  },
+  {
     id: 'name',
     Header: 'Name',
     Cell: (row) => {
       const personName = row.original.fullName
-      return personName ?
-        `${row.index + 1}. ${personName}` : ''
+      return <button className='link-button'>{personName}</button> || ''
```
Comment (ggomez-cambriasolutions) [inline app/javascript/common/search/SearchResultsTable.jsx]: can we change this className to `search-result-button`?

File: app/javascript/common/search/SearchResultsTable.jsx (hunk 1 above)
Comment (pokharelmanish) [reply app/javascript/common/search/SearchResultsTable.jsx]: removed

File: app/javascript/common/search/SearchResultsTable.jsx (hunk 2 above)
Comment (pokharelmanish) [reply app/javascript/common/search/SearchResultsTable.jsx]: done.

**Comment by pokharelmanish** [comment]
retest this please

**Pull request merged**